packages/vite/src/node/index.ts

```diff
@@ -7,7 +7,7 @@ export { formatPostcssSourceMap, preprocessCSS } from './plugins/css'
 export { transformWithEsbuild } from './plugins/esbuild'
 export { resolvePackageEntry } from './plugins/resolve'
 export { resolvePackageData } from './packages'
-export { buildErrorMessage } from './server/middlewares/error'
+export { buildErrorMessage, errorMiddleware } from './server/middlewares/error'
```

Comment:
I'm not sure about exporting the middleware, this would be the first. Usually if you need something custom, it would be better to re-implement it from scratch. There's a good chance someone might want to customize the html further in the future.

Otherwise for now perhaps you can patch `res.end` and inject the custom head tags for your usecase.

Reply:
Hi @bluwy 👋 Thank you very much for your reply 🙏. 
I completely understand your point! I just thought that this exposure could be beneficial for other frameworks to easily establish their brand as well. But of course, this is your decision in the end, and I just wanted to get your thoughts on this 😄.

Reply:
Thanks for understanding! 🙂 I think for now this could be implemented in metaframework-land, I've not seen other frameworks needing this, at most needing to customize the error overlay only.

I'll close this for now then, but it could be a possibility that we bring this up again in the future.